components/_common_/Button/__tests__/Button.test.js

```diff
@@ -64,6 +64,14 @@ describe('Button', () => {
     /* eslint-enable no-console */
   });
 
+  test('onClick prop should be called', () => {
```

Comment:
🥇 

maybe change the name to "call props.onClick when button is clicked"

Reply:
This makes more sense especially concerning your earlier comment of setting the default prop to ```() => {}```